test

```diff
@@ -6,7 +6,7 @@ if [ $? -ne 0 ]; then
 fi
 
 # integration tests
-cargo test --release --workspace -p kaspa-testing-integration --lib
+cargo nextest run --release --workspace -p kaspa-testing-integration --lib
```

Comment:
Add doctest

Reply:
We don't want doc tests here. Unless we add --full argument (or also --doc).

These tests are "pre-checks", waited on by developers before commits, so we want them to run as fast as possible and defer actual potential failures to CI. Since doc tests will be in CI, there is no reason to have them here unless manually triggered.

Reply:
It was enabled before by default. Is it time to disable it?

Reply:
Oh, sorry about the confusion.

@coderofstuff cargo test runs doc tests by default, while nextest doesn't. This means that if we switch to nextest we should add cargo test --doc --workspace as a separate step.

Reply:
This should be covered by the `cargo test --release --doc --workspace --features=no-asm` I added